templates/default/tomcat/share.xml.erb

```diff
@@ -3,7 +3,10 @@
 
     <Manager className="de.javakaffee.web.msm.MemcachedBackupSessionManager"
       memcachedNodes="<%= node['tomcat']['memcached_nodes'] %>"
-        requestUriIgnorePattern=".*\.(ico|png|gif|jpg|css|js)$"
-        transcoderFactoryClass="de.javakaffee.web.msm.serializer.kryo.KryoTranscoderFactory"
-        />
+      requestUriIgnorePattern=".*\.(ico|png|gif|jpg|css|js)$"
+      transcoderFactoryClass="de.javakaffee.web.msm.serializer.kryo.KryoTranscoderFactory"
```

Comment:
does this mean that this is mandatory?
I mean, if i don't want to install memcache, will this fail when it starts?